requirements/requirements_dev.in

```diff
@@ -1,12 +1,12 @@
 pytest>=6.2
 pytest-cov>=3.0
-pip-tools==6.4
-Sphinx<=4.1
+pip-tools>=6.4
+Sphinx<4.4
```

Comment:
Why this upper bound? Didn't we agree to remove all upper bounds?

Reply:
Unfortunately, this can't be done. I have already tried. Having Sphinx>=4.4 creates a conflict which can't be solved in python3.6 and pip-compile is not smart enough to get the right suitable version if you place a sphinx>=4.1 (or similar)

Reply:
This is utterly annoying.

Reply:
The only thing we could do is open an issue (if not yet raised) on the pop tools repo. If and when is fixed we can then integrate it in the template